### ISSUE TITLE
FadeOff leaves LED glimming

### ISSUE THREAD

**Issue by marmil** [jandelgado/jled#60]
When the FadeOff function reaches the last period step, the LED still glows as it is not fully switched off. There should be an implemented auto-off after end of fading time.

Library version: 4.5.1
Controller: ESP32-WROOM-32 at a small dev board ([AZdelivery](https://www.az-delivery.de/products/esp32-developmentboard))
My Code:

```
#include "jled.h
#define PIN_LED_INSIDE 1
auto LEDInsideOn = JLed(PIN_LED_INSIDE).LowActive();

bool schalter = true;

void setup() {
  LEDInsideOn.On();
  LEDInsideOn.Update();
  delay(500);
  LEDInsideOn.FadeOff(10000);
  LEDInsideOn.Update();
}

void loop() {
  LEDInsideOn.Update();
}
```

I dont have the original schematic by hand and can not find it in the web. It's a LowSide switched LED in principle like this;
![image](https://user-images.githubusercontent.com/1504481/96623430-a80e2780-130b-11eb-8c10-c57caa072147.png)

**Comment by jandelgado** [comment]
Hi, 

could you please provide  complete, minimal example C++ code and an schematic of the circuit you used so I can check the behaviour?

Which version of JLed are you using?

**Comment by marmil** [comment]
Hi,
thanks for your quick response.
I updated my original post with more information :)

**Comment by jandelgado** [comment]
I did some experiments and it seems like we hit this ESP32 problem: https://github.com/espressif/arduino-esp32/issues/689

The LED stays dim on in this simple example:

```
#include <jled.h>       
                                                       
auto led = JLed(22).On().LowActive();                                         
void setup() { }                                                                              

void loop() {                                                                  
  led.Update();                                                                
}              
```

I'll provide a workaround in the next days

**Comment by jandelgado** [comment]
See #61 for a fix - does it work for you?

**Comment by jandelgado** [comment]
@marmil does the fix solve the problem?

**Comment by marmil** [comment]
@jandelgado Works. Thank you 👍

**Comment by jandelgado** [comment]
fix released with 4.5.2
